manifests/o/OneTagger/OneTagger/1.5.1/OneTagger.OneTagger.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS0.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: OneTagger.OneTagger
+PackageVersion: 1.5.1
+InstallerLocale: en-US
+InstallerType: exe
+Scope: machine
+InstallModes:
+- silent
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+UpgradeBehavior: install
+ReleaseDate: 2022-10-31
+Installers:
+- Architecture: x86
```

Comment:
```suggestion
- Architecture: x64
```

Reply:
Automated validation for x86 architectures occurs on a 32-bit OS. This package installs and runs successfully on my 64-bit sandbox, but gets this error in the pipeline:
```
System.ComponentModel.Win32Exception (216): The specified executable is not a valid application for this OS platform.
```